### PR TITLE
Cache on MongoDB

### PR DESCRIPTION
**Pull request by wbruno**

### CODE REVIEW

File: lib/Mongo.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+var MongoClient = require('mongodb').MongoClient,
+    config      = require('config');
+
+/* istanbul ignore next */
+var Mongo = (function() {
+  'use strict';
+
+  var module = {
+    _init: function(callback) {
+      var url = 'mongodb://' + config.get('mongo.server') + ':' + config.get('mongo.port') + '/' + config.get('mongo.database');
```
Comment (rafaell-lycan) [inline lib/Mongo.js:10]: Can we have a config file with this params @wbruno ?

Comment (wbruno) [inline lib/Mongo.js:10]: All of that in once ? could be.

**Comment by rafaell-lycan** [comment]
#10

**Comment by rafaell-lycan** [comment]
This PR closes issue #10 

## Main changes
- Add MongoDB support
- Add new URL's V1 and V2

**Pull request merged**